src/changes/changes.xml

```diff
@@ -72,17 +72,21 @@ https://maven.apache.org/plugins/maven-changes-plugin/xsd/changes-1.0.0.xsd
     </release>
     -->
     <release version="0.17-SNAPSHOT" date="xxxx-yy-zz" description="Current SNAPSHOT - release to be done">
-      <action issue="RAT-377" type="add" dev="claudenw">
-        Added ability to specify the level of reporting on STANDARD files within a project.  This necessitated an addition
-        of a command line option "standard" to limit specify the level of detail in the STANDARD file reporting.  See command line
-        help for more details.  By default, there is no change in the reporting and only the presence of archives are reported.
+        <action issue="RAT-377" type="add" dev="claudenw">
+            Added ability to specify the level of reporting on STANDARD files within a project.  This necessitated an addition
+            of a command line option "standard" to limit specify the level of detail in the STANDARD file reporting.  See command line
+            help for more details.  By default, there is no change in the reporting and only the presence of archives are reported.
 
-        Change also fixed a major issue in license sorting.  Resulting in a change in order and expanding the name space for licenses.
-        Licenses now must have a unique id within the family name space. (e.g. family1/one is different from family2/one).
-      </action>
-      <action issue="RAT-190" type="fix" dev="claudenw">
-        Javascript (.js) files not processed as text.
-        Fixed as part of the Tika change.
+            Change also fixed a major issue in license sorting.  Resulting in a change in order and expanding the name space for licenses.
+            Licenses now must have a unique id within the family name space. (e.g. family1/one is different from family2/one).
+        </action>
+        <action issue="RAT-190" type="fix" dev="claudenw">
+            Javascript (.js) files not processed as text.
+            Fixed as part of the Tika change.
+        </action>
+        <action issue="RAT-265" type="fix" dev="claudenw">
```

Comment:
@Claudenw thanks for the fix - if I understood it correctly the old code stopped parsing filters in case of errors. Shouldn't there be a test for that behaviour, e.g. providing a list of filters with at least one problematic one?
Maybe I did not understand all available tests, but did not see a method testRAT265mixOfFiltersWithSyntaxErrors() or something similar.

Reply:
@Claudenw thanks for all your work - concerning the bugfix for RAT-265 - is there a testcase to show the correct behaviour or did I overlook it in this PR?